docs/podman-inspect.1.md

```diff
@@ -17,11 +17,12 @@ unspecified type. If a format is specified, the given template will be executed
 
 **--type, t="TYPE"**
 
-Return data on items of the specified type.  Type can be 'container', 'image' or 'all' (default: all)
+Return JSON for specified type.  Type can be 'container', 'image' or 'all' (default: all)
 
 **--format, -f="FORMAT"**
 
-Format the output using the given Go template
+Format the output using the given Go template.
+The keys of the returned JSON can be used as the values for the --format flag (see examples below).
```

Comment:
Not related to this review, but in another review @ripcurld noted that something like this:

--format "driver_name: {{.GraphDriver.Name}}"

would be valid.  Could you add an example similar to this?

Reply:
Added example